### ISSUE TITLE
an option for choose JULIA PROJECT path in `Julia`?

### ISSUE THREAD

**Issue by Roger-luo** [JuliaPy/pyjulia#338]
I'm wondering if this actually exists somewhere? It would be nice to have it, so the python package won't mess up the global environment (e.g if I'm developing something else in the global environment) and it will use a private environment instead. I notice e.g diffeqpy will load a installation script that install things in global environment directly, maybe `pyjulia` could always use a its own shared environment?

**Comment by tkf** [comment]
Did you try

```py
import os
os.environ["JULIA_PROJECT"] = "..."  # must be done before PyJulia initialization
from julia import Yao
```

?  Or `from julia import Pkg; Pkg.activate("..."')`?

Maybe you mean you want something simpler than this?

BTW, I personally don't need this because I use https://github.com/tkf/IPython.jl when developing Julia and Python.  This way, I can use Julia to controll Julia project.

**Comment by Roger-luo** [comment]
I see thanks. I thought there would be a keyword for `julia.api.Julia`
